ui/admin/tests/unit/controllers/scopes/scope/workers/new-test.js

```diff
@@ -57,14 +57,4 @@ module('Unit | Controller | scopes/scope/workers/new', function (hooks) {
 
     assert.notEqual(worker.name, 'test');
   });
-
-  test('createWorkerLed action saves changes on the specified model', async function (assert) {
```

Comment:
This test was removed because it is just a pass through for the model method `createWorkerLed` and did not do anything else.